connectivity_check.go

```diff
@@ -0,0 +1,32 @@
+package kafka
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Shopify/sarama"
+)
+
+var connectivityTimeout = 3 * time.Second
```

Comment:
If you have increased the GTG timeout, shouldn't this be also increased to the same timeout(5 sec if I remember correctly)?

Reply:
I left 3 here so we have a spare 2 seconds for the `DescribeCluster` operation (for which I guess we should decrease the timeout to fit the 5s window).

Reply:
Maybe that was the reason for the timing out on the dev cluster yesterday?